test/application/pages/address.js

```diff
@@ -4,17 +4,34 @@ module.exports = class PlaywrightDevPage {
    */
   constructor(page) {
     this.page = page;
+    this.url = "http://localhost:5010/address";
   }
 
   async continue() {
     await this.page.click(".govuk-button");
   }
 
   async goto() {
-    await this.page.goto("http://localhost:5010/address");
+    await this.page.goto(this.url);
   }
 
   async getPageTitle() {
     return await this.page.textContent("#header");
   }
+
+  isCurrentPage() {
+    return this.page.url() === this.url;
+  }
+
+  async addHouseNameOrNumber(value = "1A") {
```

Comment:
This works for now, and we can come back to using `this.user` and the `Given Name is using the system` when we start working on the other paths through the system. 👍